src/app/flashcard/components/pages/flashcard-page/flashcard-page.component.html

```diff
@@ -1,4 +1,5 @@
-<div class="view-flashcard">
-  <app-flashcard-question [flashcard]="flashcard"></app-flashcard-question>
-  <app-flashcard-answer [flashcard]="flashcard"></app-flashcard-answer>
+<h2>flashcard page</h2>
+
+<div class="view-flashcard" *ngFor="let flashcard of flashcards">
+    <div>{{flashcard.question}}</div>
```

Comment:
At a later point, this should be replaced with an actual flashcard that can take in two arguments, a question and an answer using property binding